packages/dx-react-grid-bootstrap3/src/templates/table-filter-cell.jsx

```diff
@@ -12,7 +12,7 @@ export const TableFilterCell = ({ style, filter, setFilter }) => (
       type="text"
       className="form-control"
       value={filter ? filter.value : ''}
-      onChange={e => setFilter({ value: e.target.value })}
+      onChange={e => setFilter(e.target.value ? { value: e.target.value } : null)}
```

Comment:
Should not this check be more strict? I mean the case when value is `0`. Or is the case I mention impossible?